### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by SashaViun**

### CODE REVIEW

**Review by NVyrsta — APPROVED** [review]
Good job :)

File: src/averageAges.js [hunk 1]
```diff
@@ -20,6 +20,13 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const men = people.filter(person => person.sex === 'm');
+  const age = century
+    ? men.filter(man => Math.ceil(man.died / 100) === century)
+      .map(man => man.died - man.born)
+    : men.map(man => man.died - man.born);
+
+  return age.reduce((a, b) => a + b) / age.length;
```
Comment (ArturErmolenko) [inline src/averageAges.js:29]: let us create a separate function for this repeating logic

**Review by ArturErmolenko — CHANGES_REQUESTED** [review]
GJ 👍  Just follow DRY

**Review by ArturErmolenko — APPROVED** [review]
Well done 👍

**Pull request closed without merging**